### ISSUE TITLE
Autocrafting recipe never finish, stuck in "Crafting" state

### ISSUE THREAD

**Issue by Mazlaci** [Nomi-CEu/Nomi-CEu#1137]
### Nomi CEu Version

1.7.3

### Launcher

Curseforge

### Configurations or Mods Changed

Added AE2 Fluid Crafting Rework 2.6.6 and Quark but disabled almost everything only leaving minimal mostly clientside modifications. I attached [my config file](https://github.com/user-attachments/files/17381153/quark.txt) for it. Othervise did not change the original mods or configs.

### Environment

Singleplayer

### Mode

Normal Mode

### What Happened

I made 4 wallshared EBFs intended to be used for AE2 autocrafting.
It works from the begining of the craft, using blocking mode, and after finishing the item return to my AE2 system.
But the problem is, no matter what I do, it never finishes my craft. It goes from scheduled to crafting, but stays that way even the craft finished and the item is in the AE system.
I used Interface P2P-s initially and as my last attempt, but tried round robin item conduits and item pipes as well.

### Expected Behavior

After the craft finishes, and the item enters the AE system, it registers in autocrafting menu as well.

### Reproduction Steps

I uploaded my savefile for further testing:
[2024-12-07-14-06-55.zip](https://github.com/user-attachments/files/18048510/2024-12-07-14-06-55.zip)

0. Created autocrafing recipe, put into configured EBF setup
1. Start crafing any recipe in the interface
2. Wait for item to finish, and enter AE system
3. Craft stays on

### Logs

I haven't seen anything unusual in the logs, but uplkoad them just in case.
[latest.log](https://github.com/user-attachments/files/18048497/latest.log)
[groovy.log](https://github.com/user-attachments/files/18048502/groovy.log)

**Comment by IntegerLimit** [comment]
Probably user error, check output buses, interfaces, etc.

What output bus are you using? Normal or ME?

Check if you have looped AE2 patterns, e.g. a recipe from Dust -> Ingot and a recipe from Ingot -> Dust.

**Comment by Mazlaci** [comment]
I'm using normal output busses. I checked everything and found no cicular dependecy either.
What's even wierder, after a few days later it started to work without me changing anything.

I can't reproduce it anymore either sadly, but that weekend I found it completely unsolvable.
I think at this point it might have been soma quirck at launch at the core of the issue.

I don't have any more problems with it now, even if it's still a mystery for me. So thank you for the answer. If you think so, this ticket can be closed.
